packages/expo-calendar/src/Calendar.ts

```diff
@@ -153,7 +153,7 @@ export type RecurrenceRule = {
 
 export { PermissionResponse, PermissionStatus };
 
-type OptionalKeys<T> = {
+export type OptionalKeys<T> = {
```

Comment:
Hmm, I think this is probably useless for users so I'd say we don't need to export it 😄